adam-cli/src/main/scala/org/bdgenomics/adam/cli/TransformAlignments.scala

```diff
@@ -139,6 +140,8 @@ class TransformAlignmentsArgs extends Args4jBase with ADAMSaveAnyArgs with Parqu
   var partitionByStartPos: Boolean = false
   @Args4jOption(required = false, name = "-partition_bin_size", usage = "Partition bin size used in Hive-style partitioning. Defaults to 1Mbp (1,000,000) base pairs).")
   var partitionedBinSize = 1000000
+  @Args4jOption(required = false, name = "-max_read_length", usage = "Maximum FASTQ read length, defaults to 10,000 base pairs (bp).")
+  var maxReadLength: Int = 0
```

Comment:
Should the default be 10,000?

Reply:
The default is 10,000, as set in `FastqRecordReader`.
Since it isn't possible to initialize `maxReadLength` to `null`, `0` is used instead.